pkg/cli/zip.go

```diff
@@ -113,9 +113,10 @@ func newZipper(f *os.File) *zipper {
 	}
 }
 
-func (z *zipper) close() {
-	_ = z.z.Close()
-	_ = z.f.Close()
+func (z *zipper) close() error {
+	err1 := z.z.Close()
+	err2 := z.f.Close()
+	return errors.CombineErrors(err1, err2)
```

Comment:
That feeling when a library is well-designed 😍

Reply:
:)